assets/js/googlesitekit/notifications/register-defaults.js

```diff
@@ -559,6 +561,54 @@ export const DEFAULT_NOTIFICATIONS = {
 			);
 		},
 	},
+	'web-data-stream-not-available-notification': {
+		Component: WebDataStreamNotAvailableNotification,
+		priority: 10,
+		areaSlug: NOTIFICATION_AREAS.BANNERS_ABOVE_NAV,
+		viewContexts: [ VIEW_CONTEXT_MAIN_DASHBOARD ],
+		isDismissible: false,
```

Comment:
This is the reason your dismissal was probably not working 😄 This should be true for the datastore dismissal logic to kick in.